docs/source/modelbuilding.rst

```diff
@@ -13,6 +13,19 @@ Variables created in a ``setup`` method are added to the model even if they are
 
 At the end of this page a detailed example shows this technique in practice.
 
+Accessing Variables in Models
```

Comment:
Nice. I really like this method of documenting things; I think we should convert much of the rest of the documentation to this style.